src/helpers/swap.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+function swap(&$a, &$b)
+{
+    $temp = $a;
+    $a = $b;
+    $b = $temp;
```

Comment:
Given that Laravel 5.7+ only supports PHP 7.1+, the entire body of this function can be simplified to `[$a, $b] = [$b, $a];`

Reply:
I actually googled it before I did this PR. Someone did benchmarks and it's around twice as slow, so I figured I'd just be old fashioned.

https://stackoverflow.com/questions/3541730/is-there-a-php-function-for-swapping-the-values-of-two-variables

```
<?php

function stopwatch($callback, $times = 1)
{
    $totalTime = 0;

    foreach (range(1, $times) as $time) {
        $start = microtime(true);
        $callback();
        $totalTime += microtime(true) - $start;
    }

    return $totalTime / $times;
}

function swap1(&$a, &$b)
{
	[$a, $b] = [$b, $a];
}

function swap2(&$a, &$b)
{
	$temp = $a;
	$b = $a;
	$a = $temp;
}

echo stopwatch(function () { $a = 1; $b = 2; swap1($a, $b); }, 100) . PHP_EOL;
echo stopwatch(function () { $a = 1; $b = 2; swap2($a, $b); }, 100) . PHP_EOL;
// Result:
// 5.6266784667969E-7
// 2.0980834960938E-7
```

Reply:
Fair enough, then.